spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -104,6 +108,40 @@ def _prod(iterable):
     return reduce(operator.mul, iterable, 1)
 
 
+def _all_gen(rd):
```

Comment:
Both this and the following function seem like they should be somewhere more general (utility_calls perhaps?) than in the population vertex.  I don't mind them being here for now but perhaps consider moving them somewhere else in the future?

Reply:
Yes, could be moved if needed in future; currently they are only ever used here...